lib/sensors/thermistor.cpp

```diff
@@ -0,0 +1,27 @@
+#include "mbed.h"
+#include "thermistor.h"
+
+thermistor::thermistor(PinName thermistor_vout_pin): thermistor_voltage_(thermistor_vout_pin);
+
+float thermistor::readThermistor()

```

Comment:
change functuon name to readThermistorDegrees

Reply:
Should I specify celcius/kelvin in the name as well?

Reply:
oh crap forgot both can be called degrees...maybe readThermistorCelcius then